static/app/views/traces/content.tsx

```diff
@@ -386,10 +412,12 @@ function SpanTable({
 }
 
 function SpanRow({
+  organization,
   span,
   trace,
   setHighlightedSliceName,
 }: {
+  organization: Organization;
```

Comment:
Any particular reason we didn't just put `useOrganization` here instead of passing it down?

Reply:
This is rendered for every row, I moved it to the parent Table instead so we only do it once.